README.md

```diff
@@ -1,8 +1,107 @@
 # ActiveRecord Associations in Sinatra
 
-##TEACHER OUTLINE
-+ Creating multiple tables
-+ Describe has many/belongs to relationships
-+ Setting up has_many, belongs_to relationship between models
-+ Discuss the use of primary keys as the link between models in ActiveRecord
-+ Should allude to other types of relationships (many to many, one to one, etc) but not discuss join tables.
\ No newline at end of file
+## Objectives
+
+1. Understand primary and foreign key.
+2. Understand the has many/ belongs to relationship.
+
+
```

Comment:
I think we need a little more opening exposition. like "We learned about foreign and primary keys in sql and how they relate tables to one another - we're going to learn how to do this using ActiveRecord"